automation_testing/utils/data/dataview_data.ts

```diff
@@ -1,4 +1,12 @@
-const dataview = {
+const dataview_data = {
```

Comment:
Please pull from develop and push again. This file has already been reviewed in another PR.